src/django_csrf_fetch.js

```diff
@@ -87,22 +76,17 @@ const handleEmptyJSON = json => (
  *  - response JSON is returned in place of response
  */
 export const fetchJSONWithCSRF = async (input: string, init: Object = {}): Promise<*> => {
-  let response = await fetch(input, formatJSONRequest(init))
-  let text = await response.text()
+  const response = await fetch(input, formatJSONRequest(init))
+  const text = await response.text()
+
+  const jsonFriendlyText = handleEmptyJSON(text)
+  const json = JSON.parse(jsonFriendlyText)
```

Comment:
Can we just use `response.json()` here instead?

Reply:
I think we wanted to handle empty responses with a `{}`, but I can't remember why we wanted to do that. I'm fine with switching to `.json()` if you are

Reply:
I think it's probably the cleaner approach. If it's an empty response, do you know what's returned by `.json()`?

Reply:
I tested it and it converts it to an empty string, which I think makes sense

Reply:
yeah, the only issue I see with that is that we'd be calling `...""`, which in the context of an object literal wouldn't make much sense. So we maybe want to do something:

```js
let json = await response.json()
json = json === ""
  ? {}
  : json
```

or something along those lines, to ensure we're always going to be `...`-ing an object